docs/en/ingest-management/release-notes/release-notes-8.7.asciidoc

```diff
@@ -0,0 +1,222 @@
+:agent-pull: https://github.com/elastic/elastic-agent/pull/
+:fleet-server-issue: https://github.com/elastic/fleet-server/issues/
+:fleet-server-pull: https://github.com/elastic/fleet-server/pull/
+
+[[release-notes]]
+= Release notes
+
+This section summarizes the changes in each release.
+
+* <<release-notes-8.7.0>>
+
+Also see:
+
+* {kibana-ref}/release-notes.html[{kib} release notes]
+* {beats-ref}/release-notes.html[{beats} release notes]
+
+// begin 8.7.0 relnotes
+
+[[release-notes-8.7.0]]
+== {fleet} and {agent} 8.7.0
+
+Review important information about the {fleet} and {agent} 8.7.0 release.
+
+[discrete]
+[[breaking-changes-8.7.0]]
+=== Breaking changes
+
+Breaking changes can prevent your application from optimal operation and
+performance. Before you upgrade, review the breaking changes, then mitigate the
+impact to your application.
+
+[discrete]
+[[breaking-147616]]
+.Remove the current_upgrades endpoint
+[%collapsible]
+====
+*Details* +
+The `api/fleet/current_upgrades` endpoint has been removed. For more information, refer to {kibana-pull}147616[#147616].
+*Impact* +
+When you upgrade to 8.7.0, use the `/action_status` endpoint.
+====
+
+[discrete]
+[[breaking-147199]]
+.Remove the preconfiguration API route
+[%collapsible]
+====
+*Details* +
+The `/api/fleet/setup/preconfiguration` API, which was released as generally available by error, has been removed. For more information, refer to {kibana-pull}147199[#147199].
+*Impact* +
+Do not use `/api/fleet/setup/preconfiguration`. To manage preconfigured agent policies, use `kibana.yml`. For more information, check link:https://www.elastic.co/guide/en/kibana/current/fleet-settings-kb.html#_preconfiguration_settings_for_advanced_use_cases[Preconfigured settings].
+====
+
+[discrete]
+[[deprecations-8.7.0]]
+=== Deprecations
+
+The following functionality is deprecated in 8.7.0, and will be removed in
+8.7.0. Deprecated functionality does not have an immediate impact on your
+application, but we strongly recommend you make the necessary updates after you
+upgrade to 8.7.0.
+
+{agent}::
+* add info
+
+[discrete]
+[[new-features-8.7.0]]
+=== New features
+
+The 8.7.0 release adds the following new and notable features.
+
+{fleet}::
+* Add ability to show FQDN of agents {kibana-pull}150239[#150239]
+* Add `getStatusSummary` query parameter to `GET /api/fleet/agents` API {kibana-pull}149963[#149963]
+* Enable diagnostics feature flag and change query for files to use upload_id {kibana-pull}149575[#149575]
+* Add experimental toggles for doc-value-only {kibana-pull}149131[#149131]
+* Display agent metrics, CPU and memory in the agent list table and agent details page {kibana-pull}149119[#149119]
+* Implement subcategories in integrations UI {kibana-pull}148894[#148894]
+* Add rollout period to upgrade action {kibana-pull}148240[#148240]
+* Add per-policy inactivity timeout + use runtime fields for agent status {kibana-pull}147552[#147552]
```

Comment:
Can you replace + with and here?

Reply:
Addressed in [b3a983d](https://github.com/elastic/observability-docs/pull/2788/commits/b3a983d65af3b2d0495f85fb06f787fd1bca0ec0)